docs/docusaurus/docs/core/expectation_suites/manage_expectation_suites.md

```diff
@@ -0,0 +1,294 @@
+---
+sidebar_label: 'Manage Expectation Suites'
+title: 'Manage Expectation Suites'
+description: Create and manage sets of Expectations in Python with GX Core.
+---
+
+An Expectation Suite contains a group of Expectations that describe the same set of data.  All of the Expectations that you apply to your data will grouped into Expectation Suites.
+
+## Prerequisites
+
+This guide assumes you have:
+
+- Installed Python.
+- Installed the GX Core library.
+
+## Create an Expectation Suite
+
+1. Import the GX Core library and the `ExpectationSuite` class.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py imports"
+  ```
+
+2. Get a Data Context.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py get_context"
+  ```
+
+3. Create an Expectation Suite.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py create Expectation Suite"
+  ```
+
+4. Add the Expectation Suite to your Data Context.
+  ```python name="core/expectation_suites/_examples/create_an_expectation_suite.py add snippet to Data Context"
+  ```
+
+:::tip
```

Comment:
```suggestion
:::tip
```
This might be better as an optional step. If this is unclear, we can discuss it.

Reply:
My thought is that this qualifies as a "tip" because it presents an optional way to consolidate the steps that were performed above.  It isn't an optional step, per say --  rather, it is an optional way of performing the previously described steps in fewer lines of code.